package.json

```diff
@@ -52,6 +52,8 @@
   "dependencies": {
     "@chainlink/contracts": "^0.0.5",
     "@openzeppelin/contracts": "2.5.1",
+    "@openzeppelin/contracts-ethereum-package": "2.5.0",
+    "@openzeppelin/upgrades": "^2.8.0",
```

Comment:
Should we use strict version number here? 2.8.0

Reply:
I believe they are on 3 now for solc 6 but yea couldn't hurt to make it strict.